app.json

```diff
@@ -16,7 +16,7 @@
                 "DEPLOYMENT": "Heroku Review App",
                 "AUTO_LOGIN": true
             },
-            "buildpacks": [{ "url": "heroku-community/apt" }, { "url": "heroku/nodejs" }, { "url": "heroku/python" }],
+            "buildpacks": [{ "url": "heroku/nodejs" }, { "url": "heroku/python" }],
```

Comment:
We no longer attempt to install `xmlsec` deps on Heroku. SAML won't run on Heroku (documented).